### PR TITLE
Add examples for Opta

### PR DESCRIPTION
**Pull request by manchoz**
Adding examples for Arduino Opta Lite, RS485 and WiFi to AWS IoT Core section.

This is needed for getting the Opta pass the AWS DA Qualification.

The sketches use modern Arduino libraries to manage connectivity.

### CODE REVIEW

File: examples/AWS IoT/AWS_IoT_Opta/AWS_IoT_Opta_WiFi/AWS_IoT_Opta_WiFi.ino [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+// Remove comment for using the Ethernet connection
+// #define USE_ETHERNET_CONNECTION
+
+#if defined(USE_ETHERNET_CONNECTION)
+#include <Ethernet.h>
+#include <EthernetUdp.h>
+EthernetConnectionHandler conMan;
+EthernetClient tcpClient;
+EthernetUDP NTPUdp;
+#else
```
Comment (pennam) [inline examples/AWS IoT/AWS_IoT_Opta/AWS_IoT_Opta_WiFi/AWS_IoT_Opta_WiFi.ino]: If the example is WiFi only i would remove the Ethernet variables, or we can add one single example working both for ethernet and WiFi and name it  `AWS_IoT_Opta.ino`

**Review by pennam — CHANGES_REQUESTED** [review]
Do we need the opta examples to be standalone? I think the WiFi one can be merged with this [AWS_IoT_WiFi.ino](https://github.com/arduino/ArduinoCloudProviderExamples/blob/master/examples/AWS%20IoT/AWS_IoT_WiFi/AWS_IoT_WiFi.ino) 

If it is not possible to merge the examples i would propose to rename also the existing one to  `AWS_IoT_MKR_WiFi.ino` or something similar.

File: examples/AWS IoT/AWS_IoT_Opta/AWS_IoT_Opta_WiFi/AWS_IoT_Opta_WiFi.ino (hunk 1 above)
Comment (manchoz) [reply examples/AWS IoT/AWS_IoT_Opta/AWS_IoT_Opta_WiFi/AWS_IoT_Opta_WiFi.ino]: We need standalone examples for getting the AWS DA Qualification.

I agree to remove the Ethernet stuff for the WiFI example.
I also agree to rename the other examples to `MKR`.

Comment (manchoz) [reply examples/AWS IoT/AWS_IoT_Opta/AWS_IoT_Opta_WiFi/AWS_IoT_Opta_WiFi.ino]: Ethernet configuration removed

**Review by manchoz — COMMENTED** [review]
Ethernet configuration removed from WiFi example

**Pull request merged**